### DISCUSSION TITLE
Email ticket generation setup

### DISCUSSION THREAD

**Discussion by Odiesinfosec** [Peppermint-Lab/peppermint#168] [Q&A]
So, I can see in the commit history and on the latest release, there are  several references to "mail"
It is marked as complete in issue #162 yet there isnt any documentation on how to set that up.

to clarify; 
Does peppermint support email ticket generation and reply via email?
if so, 
How can we configure that?

**Comment by Odiesinfosec** [comment]
Oh and I have made hundreds of KB articles for the most inept end users, Id love to help document this and give back

**Reply by potts99** [reply]
IMAP mailbox listening is now available on latest tag, set up in admin area

**Reply by potts99** [reply]
KB articles around peppermint? Or in general

**Reply by adheerp** [reply]
Is there any article for setting up the peppermint email logging system?

**Comment by JayJayonGeethub** [comment]
Was this ever addressed? i feel like i am fumbling on the same questions over and over again.

**Reply by adheerp** [reply]
I didn't find anything related to this. I don't know if people have used this feature or not.

**Comment by JayJayonGeethub** [comment]
That’s so strange as to not use that feature . Otherwise how does a tech
know when a new world we has arrived?
